### PR TITLE
add ChEMBL search tool

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

File: chemicaltoolbox/chembl/chembl.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
+from chembl_webresource_client.new_client import new_client
+import argparse
+
+def open_file(filename):
+    with open(filename) as f:
+        return f.read().split('\n')[0]
```
Comment (bgruening) [inline chemicaltoolbox/chembl/chembl.py]: Can you replace this function by `file.readline()`?

File: chemicaltoolbox/chembl/chembl.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    """
+    Return only compounds which are approved drugs
+    """
+    return mols.filter(max_phase=4)
+
+def filter_biotherapeutic(mols):
+    """
+    Return only biotherapeutic molecules
+    """
+    return mols.filter(biotherapeutic__isnull=False)
```
Comment (bgruening) [inline chemicaltoolbox/chembl/chembl.py]: is it really two `__`?

File: chemicaltoolbox/chembl/chembl.xml [hunk 3]
```diff
@@ -0,0 +1,118 @@
+<tool id="chembl" name="Search ChEMBL database" version="0.1.0">
+    <description>for compounds which are similar to a SMILES string</description>
+    <requirements>
+        <requirement type="package" version="0.9.31">chembl_webresource_client</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python -W ignore '$__tool_directory__/chembl.py'
```
Comment (bgruening) [inline chemicaltoolbox/chembl/chembl.xml]: What does this `-W ignore` filter?

File: chemicaltoolbox/chembl/chembl.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+def open_file(filename):
+    with open(filename) as f:
+        return f.read().split('\n')[0]
+
+def get_smiles(res):
+    """
+    Get a list of SMILES from function results
+    """ 
+    smiles = []
```
Comment (bgruening) [inline chemicaltoolbox/chembl/chembl.py]: Should that be a set to exclude duplicates?

File: chemicaltoolbox/chembl/chembl.xml (hunk 3 above)
Comment (simonbray) [reply chemicaltoolbox/chembl/chembl.xml]: The python package raises a warning which I'd like to hide from the user. That flag suppresses it.

File: chemicaltoolbox/chembl/chembl.py (hunk 2 above)
Comment (simonbray) [reply chemicaltoolbox/chembl/chembl.py]: yes...

File: chemicaltoolbox/chembl/chembl.py (hunk 1 above)
Comment (simonbray) [reply chemicaltoolbox/chembl/chembl.py]: Will do.

File: chemicaltoolbox/chembl/chembl.py (hunk 4 above)
Comment (simonbray) [reply chemicaltoolbox/chembl/chembl.py]: changed

**Comment by bgruening** [comment]
Merci!

**Pull request merged**